tests/unit_tests/test_subtensor.py

```diff
@@ -1914,6 +1914,24 @@ def test_get_subnets_no_block_specified(mocker, subtensor):
     subtensor.query_map_subtensor.assert_called_once_with("NetworksAdded", None)
 
 
+def test_get_subnets_correct_length(mocker, subtensor):
+    """Test get_subnets returns a list of the correct length."""
+    # Prep
+    block = 123
+    num_records = 50
```

Comment:
Shouldn't this test for length > 100?

Reply:
You're right—I missed a zero. I've added it and pushed a changed version along with the updated test, but the test still doesn't catch the issue. I also updated get_subnets again and verified it by running your snippet.

        
```python
import bittensor

sub = bittensor.subtensor("wss://test.chain.opentensor.ai:443")
subnets = sub.get_subnets()
n_subnets = sub.get_total_subnets()

len(subnets) == n_subnets
# True; Expected: True
```